src/PoolTogetherGovernor.sol

```diff
@@ -0,0 +1,162 @@
+
+import {
+  Governor, GovernorCountingFractional
+} from "flexible-voting/src/GovernorCountingFractional.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {ICompoundTimelock} from
+  "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+import {IPoolTogetherTimelock} from "src/interfaces/IPoolTogetherTimelock.sol";
+import {GovernorTimelockCompound} from "src/lib/GovernorTimelockCompound.sol";
+
+/// @notice The upgraded PoolTogether Governor: Bravo compatible and built with OpenZeppelin.
+contract PoolTogetherGovernor is
+  GovernorCountingFractional,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the POOL token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant POOL_TOKEN =
+    ERC20VotesComp(0x0cEC1A9154Ff802e7934Fc916Ed7Ca50bDE6844e);
+
+  /// @notice The address of the existing PoolTogether DAO Timelock on Ethereum mainnet through
+  /// which
+  /// this Governor executes transactions.
+  IPoolTogetherTimelock private constant TIMELOCK =
+    IPoolTogetherTimelock(payable(0x42cd8312D2BCe04277dD5161832460e95b24262E));
+
+  /// @notice Human readable name of this Governor.
+  string private constant GOVERNOR_NAME = "PoolTogether Governor Bravo";
+
+  /// @notice The number of POOL (in "wei") that must participate in a vote for it to meet quorum
+  /// threshold.
+  ///
+  /// TODO: placeholder
+  uint256 private constant QUORUM = 100_000e18; // 100,000 POOL
+
+  /// @param _initialVotingDelay The deployment value for the voting delay this Governor will
+  /// enforce.
+  /// @param _initialVotingPeriod The deployment value for the voting period this Governor will
+  /// enforce.
+  /// @param _initialProposalThreshold The deployment value for the number of POOL required to
+  /// submit
+  /// a proposal this Governor will enforce.
+  constructor(
+    uint256 _initialVotingDelay,
+    uint256 _initialVotingPeriod,
+    uint256 _initialProposalThreshold
+  )
+    GovernorVotesComp(POOL_TOKEN)
+    GovernorSettings(_initialVotingDelay, _initialVotingPeriod, _initialProposalThreshold)
+    GovernorTimelockCompound(TIMELOCK)
+    Governor(GOVERNOR_NAME)
+  {}
+
+  /// @dev We override this function to resolve ambiguity between inherited contracts.
+  function supportsInterface(bytes4 interfaceId)
+    public
+    view
+    virtual
+    override(Governor, GovernorTimelockCompound)
+    returns (bool)
+  {
+    return GovernorTimelockCompound.supportsInterface(interfaceId);
+  }
+  /// @dev We override this for it to show up in our scopelint spec
```

Comment:
Hmm this might be worth deleting at this point, in the interest of minimizing the lines of code / potential sources of error here. Feels a bit hacky to have production code that only exists to cater to test tools. I'd say remove this, and update the README to include "steps for using scopelint spec", with adding this method as one of the steps

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/20/commits/a8521b0f5f718aea46818b6818b825070be4975c

Reply:
@alexkeating I unresolved this one since I don't see instructions in the README for how to add this back to get an accurate `scopelint spec` report, I think this would be good to document

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/20/commits/163446f924479827f72d255987fdbdbe703345f5